src/components/resourceSelector.js

```diff
@@ -3,6 +3,7 @@ import { useStaticQuery } from "gatsby"
 import Partial from "./partial"
 import propTypes from "prop-types"
 import withLocation from "./withLocation"
+import { graphql } from "gatsby"
```

Comment:
this was suggested (not required) by a warning I was getting:

```
warn Using the global `graphql` tag is deprecated, and will not be supported in v3.
Import it instead like:  import { graphql } from 'gatsby' in file:
/Users/workward/projects/documentation/src/components/resourceSelector.js
```

Reply:
This warning is coming from other files as well, but it doesn't always specify where like it does here.